benchmarks/release_benchmarking/release/v2.6.0/v2.6.0.md

```diff
@@ -0,0 +1,156 @@
+# HoloHub Benchmarks - Holoscan SDK v2.6.0
+
+## Background
+
+Holoscan SDK emphasizes low end-to-end latency in application pipelines. The following benchmarks
+reflect latency measures in maintained HoloHub applications built with
+[Holoscan SDK release v2.6.0](https://github.com/nvidia-holoscan/holoscan-sdk/releases/tag/v2.6.0).
+
+## Platform Configurations
+
+We collected benchmark data on the following platforms:
+
+|         | x86_64 | IGX iGPU | IGX dGPU (A6000) | IGX dGPU (ADA6000) |
+| ------- | ------ | -------- | -------- | -------- |
+| OS      | Ubuntu 22.04.3 | Ubuntu 22.04.4 LTS | Ubuntu 22.04.4 | Ubuntu 22.04.4 |
+| CPU     | Intel(R) Core(TM) i9-10980XE CPU @ 3.00GHz | Cortex-A78AE | Cortex-A78AE | Cortex-A78AE |
+| Discrete GPU | NVIDIA RTX A6000 | Orin (nvgpu) | NVIDIA RTX A6000 | NVIDIA RTX ADA6000 |
+| NVIDIA Display Driver | 535.183.01 | [N/A] | 535.171.04 | 535.171.04 |
+| CUDA Version | 12.6 | 12.6 | 12.6 | 12.6 |
```

Comment:
```suggestion
| CUDA Version | 12.6 | 12.2 | 12.2 | 12.2 |
```

Reply:
Thanks for catching this. Agreed that CUDA version here intends to reflect driver capabilities rather than the CUDA Toolkit in the environment. Applying your suggestion and also reverting x86 to match 535 drivers:

```suggestion
| CUDA Version | 12.2 | 12.2 | 12.2 | 12.2 |
```